### DISCUSSION TITLE
scan scripts not executing anymore

### DISCUSSION THREAD

**Discussion by JudsonHat** [nzbgetcom/nzbget#271] [General]
I have noticed that my two scan scripts have stopped being executed.  I don't know when they stopped working.  One of them is shown below.

Did anything get changed that would cause this script to stop functioning?  Or does someone see why it wouldn't work?  I've double checked that the script is enabled in the category, and I don't see anything relevant in the logs.

```
#!/bin/bash

##############################################################################
### NZBGET SCAN SCRIPT                                                     ###

# 
# Set the dupekey to the lowercase nzb-name.
#

### NZBGET SCAN SCRIPT                                                     ###
##############################################################################

echo "[INFO] #####################################"
echo "[INFO] DUPEKEY SCRIPT"

[ $NZBNP_DUPEKEY ] && echo "[INFO] DUPEKEY already present" && exit 93

echo "[NZB] DUPEKEY=${NZBNP_NZBNAME,,}" && echo "[INFO] Success" && exit 93 || echo "[ERROR] Failed";
exit 94
```

**Comment by dnzbk** [comment]
I tested your extension and it works for me.
Could you make sure that the extension is activated (play button) in the extension manager?

**Comment by JudsonHat** [comment]
Thank you!

Actually, none of my scripts (scan scripts or otherwise) were activated.  I just relied on them being enabled for each Category.  All of my post process scripts were being executed as expected, just not my three scan scripts.

But once I activated just one of my scan scripts, all 3 started working in each Category they were enabled.  Strange.  Must be some kind of bug.

Anyway, thank you for your input leading me to getting it resolved!
